Client/Shared/UserProfile/ProfileDetails.razor

```diff
@@ -0,0 +1,96 @@
+﻿@if (_userProfileDetails is null)
+{
+    <LoadingIndicatorComponent />
+}
+else
+{
+    @if (_userProfileDetails.TopUsersPlacement is not null)
+    {
+        <span class="top-user @($"top-{_userProfileDetails.TopUsersPlacement}-user")">
+            <i class="icon-cup"></i>
+            @switch (_userProfileDetails.TopUsersPlacement)
+            {
+                case 1:
```

Comment:
Wrong indexing 
![image](https://user-images.githubusercontent.com/56700157/134816842-fafa7cea-6c0e-4b10-9aa0-24f5e6e523e3.png)
![image](https://user-images.githubusercontent.com/56700157/134816849-fb673869-1349-41f5-bb56-198fc5e8c006.png)

Reply:
Also there is no ';' in project